src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form form_big"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <label class="form__label">
+        <input
+        class="form__input form__input_big"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
```

Comment:
code is not aligned
check the checklist 
```suggestion
      <label class="form__label">
        <input
          class="form__input form__input_big"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
      >
```